### ISSUE TITLE
Batch Normalization in Caffe?

### ISSUE THREAD

**Issue by AntreasAntoniou** [NVIDIA/caffe#29]
Hello guys,

I was wondering if there is a batch normalization layer available for Caffe? I looked the documentation and couldn't find anything. In google I found some custom made versions of it, but I haven't used git for long so I don't know whether adding those would mess up my installation. Anyone have any hints?

**Comment by lukeyeager** [comment]
See 
BVLC/caffe#1965
BVLC/caffe#

**Comment by lukeyeager** [comment]
The batch_norm layer is included in v0.14.

**Comment by AntreasAntoniou** [comment]
Wow, thanks for the follow up man.
